sematic/db/models/resolution.py

```diff
@@ -199,6 +199,7 @@ class Resolution(Base, JSONEncodableMixin, HasExternalJobsMixin):
     container_image_uri: Optional[str] = Column(types.String(), nullable=True)
     client_version: Optional[str] = Column(types.String(), nullable=True)
     cache_namespace: Optional[str] = Column(types.String(), nullable=True)
+    user_id: Optional[str] = Column(types.String(), nullable=True)
```

Comment:
Please see [this comment](https://github.com/sematic-ai/sematic/pull/602#discussion_r1117467892).

Reply:
Done.